### PR TITLE
feat: xcm status flow

### PR DESCRIPTION
**Pull request by Victor-Salomon**
Add Ongoing transfer state and update status message
Capture d’écran 2024-12-04 à 12 40 04
Capture d’écran 2024-12-04 à 12 40 14

### CODE REVIEW

File: app/src/models/transfer.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ export interface StoredTransfer extends RawTransfer {
   sendResult?: toEthereum.SendResult | toPolkadot.SendResult
   // A subscan unique Id shared accross chains to track ongoing transfers
   uniqueTrackingId?: string
+  // Transfer state
+  state?: string
```
Comment (NunoAlexandre) [inline app/src/models/transfer.ts:37]: It should be 'status' 👍 
```suggestion
  status?: string
```

**Review by NunoAlexandre — APPROVED** [review]
Great :) just a name comment but it's good to go!

File: app/src/utils/transferTracking.ts [hunk 2]
```diff
@@ -130,13 +130,13 @@ export const getTransferStatusToPolkadot = (txTrackingResult: FromEthTrackingRes
...
     case TransferStatus.Complete:
-      return 'Transfer completed'
+      return 'Completed'
 
     case TransferStatus.Failed:
-      return 'Transfer Failed'
+      return 'Failed'
 
     default: // Should never happen
       return 'Unknown'
```
Comment (noahjoeris) [inline app/src/utils/transferTracking.ts:142]: To be consistent.
```suggestion
      return 'Unknown status'
```

**Review by noahjoeris — APPROVED** [review]
Nice nice nice nice nice nice nice nice nice nice nice nice nice nice nice nice nice ❤️

**Comment by NunoAlexandre** [comment]
@Victor-Salomon testing this locally, indeed we should bump the average XCM duration to 2 or 2.5 to balance on the one hand the progress bar with the status updates and on the other hand the total duration it takes to mark the tx as completed (close to 3 min).

File: app/src/hooks/useTransferProgress.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { Direction } from '@/services/transfer'
 const ESTIMATED_DURATION = {
   toEthereum: 4 * 60 * 60, // 30 mins in seconds
   toPolkadot: 30 * 60, // 4h in seconds
-  xcmTransfer: 3 * 60, // 3 mins in seconds
+  xcmTransfer: 1.5 * 60, // 1.5 minutes in seconds
```
Comment (NunoAlexandre) [inline app/src/hooks/useTransferProgress.tsx:8]:
```suggestion
  xcmTransfer: 2.5 * 60, // 2.5 minutes in seconds
```

**Comment by NunoAlexandre** [comment]
@Victor-Salomon updated!

Let's wait for a successful build and then merge :)

**Pull request merged**